test/Microsoft.AspNet.Mvc.Razor.Test/LanguageViewLocationExpanderTest.cs

```diff
@@ -14,6 +14,26 @@ public static IEnumerable<object[]> ViewLocationExpanderTestDataWithExpectedValu
             {
                 yield return new object[]
                 {
+                    LanguageViewLocationExpanderOption.Suffix,
+                    new[]
+                    {
+                        "/Views/{1}/{0}.cshtml",
+                        "/Views/Shared/{0}.cshtml"
+                    },
+                    new[]
+                    {
+                        "/Views/{1}/en-GB.{0}.cshtml",
```

Comment:
This doesn't seem like a suffix - shouldn't it be `/Views/Home/Index.en.cshtml` as opposed to `/Views/Home/en.Index.cshtml`?